ntoskrnl/include/internal/arm/mm.h

```diff
@@ -91,9 +91,16 @@
 #define MI_MAKE_WRITE_PAGE(x)      ((x)->u.Hard.ReadOnly = 0)
 
 /* Macros to identify the page fault reason from the error code */
-#define MI_IS_NOT_PRESENT_FAULT(FaultCode) TRUE
-#define MI_IS_WRITE_ACCESS(FaultCode) TRUE
-#define MI_IS_INSTRUCTION_FETCH(FaultCode) FALSE
+#define MI_IS_NOT_PRESENT_FAULT(FaultCode)  TRUE
+#define MI_IS_WRITE_ACCESS(FaultCode)       TRUE
+// 0x00000004: user-mode access.
+// TODO: Confirm which value to use.
+#define MI_IS_RESERVED_BIT_FAULT(FaultCode) FALSE
+#define MI_IS_INSTRUCTION_FETCH(FaultCode)  FALSE
+// 0x00000020: protection-key violation.
+// Bits 6-14: reserved.
+// 0x00008000: violation of SGX-specific access-control requirements.
+// Bits 16-31: reserved.
```

Comment:
These bits are x86-specific, so this is nonsensical.

Reply:
Right.
Replaced with a better TODO.